orchagent/port.h

```diff
@@ -55,6 +57,8 @@ class Port
     int                 m_ifindex = 0;
     sai_object_id_t     m_port_id = 0;
     sai_object_id_t     m_vlan_oid = 0;
+    sai_object_id_t     m_bridge_id = 0;
```

Comment:
i will remove this line.

---
In reply to: [126868296](https://github.com/Azure/sonic-swss/pull/250#discussion_r126868296) [](ancestors = 126868296,126867393,126781483)